src/vivarium_census_prl_synth_pop/components/person.py

```diff
@@ -119,7 +126,10 @@ def on_time_step(self, event: Event) -> None:
             event.index,
             query="alive == 'alive'",
         )
-
+        # Want all addresses, including from dead simulants
+        address_id_to_state_puma_map = self._get_non_gq_address_id_to_state_puma_mapping(
+            self.population_view.get(event.index)
```

Comment:
I don't know Vivarium well enough to know whether it matters, but would it be better here to call `self.population_view.get(...)` only once and do the alive filtering outside of `query`?

Reply:
Yeah, probably, good call